helmchart/sdp/minimal-values.yaml

```diff
@@ -0,0 +1,69 @@
+
+## @section Minimal Values
+## @descriptionStart
+##  These are the minimal values required to deploy the SDP helm chart.
+## @descriptionEnd
+
+sdp:
+  ## @param sdp.configMap.data.EC256_PUBLIC_KEY [string] The EC256 public key used for authentication purposes.
+  ## @param sdp.configMap.data.SEP10_SIGNING_PUBLIC_KEY Anchor platform SEP10 signing public key.
+  ## @param sdp.configMap.data.DISTRIBUTION_PUBLIC_KEY The public key of the Stellar distribution account that sends the Stellar payments.
+  ## @param sdp.configMap.data.RECAPTCHA_SITE_KEY Site key for ReCaptcha. Required if using ReCaptcha.
+  configMap:
+    annotations:
+    data:
+      EC256_PUBLIC_KEY: #required
+      SEP10_SIGNING_PUBLIC_KEY: #required
+      DISTRIBUTION_PUBLIC_KEY: #required
+      RECAPTCHA_SITE_KEY: #required
+
+  ## @param sdp.kubeSecrets.data.EC256_PRIVATE_KEY [string] The EC256 Private Key. This key is used to sign the authentication token.
+  ## @param sdp.kubeSecrets.data.SEP10_SIGNING_PRIVATE_KEY The public key of the Stellar account that signs the SEP-10 transactions. It's also used to sign URLs.
+  ## @param sdp.kubeSecrets.data.SEP24_JWT_SECRET The JWT secret that's used by the Anchor Platform to sign the SEP-24 JWT token.
+  ## @param sdp.kubeSecrets.data.RECAPTCHA_SITE_SECRET_KEY Secret key for Google reCAPTCHA service to verify user's non-robotic behavior.
+  ## @param sdp.kubeSecrets.data.ANCHOR_PLATFORM_OUTGOING_JWT_SECRET The JWT secret used to create a JWT token used to send requests to the anchor platform.
+  ## @param sdp.kubeSecrets.data.DATABASE_URL URL of the database used by the SDP.
+  ## @param sdp.kubeSecrets.data.DISTRIBUTION_SEED The private key of the Stellar account used to disburse funds. This is needed for the init container
+  kubeSecrets:
+    create: true
+    data:
+      EC256_PRIVATE_KEY: #required
+      SEP10_SIGNING_PRIVATE_KEY: #required
+      SEP24_JWT_SECRET: #required
+      RECAPTCHA_SITE_SECRET_KEY:  #required
+      ANCHOR_PLATFORM_OUTGOING_JWT_SECRET: #required for mySdpToAnchorPlatformSecret
+      DATABASE_URL: #required
+      DISTRIBUTION_SEED: #required
```

Comment:
I don't believe it would be a problem if people create k8s secrets by populating this file and then installing the helm chart to their kube cluster. As long as they don't commit this file to source control, which I don't believe is the intent. 

IMO `kubeSecrets.secretName` is the alternative. The only reason anyone would use that is if they have an external system creating secrets like ESO in our case. 

The rest they'll either create the secrets manually and rarely update them in releases, or they'll have their CI pipeline fill a tokenized version of this minimum-values.yaml to publish the secrets (from Vault/Consul or from secrets stored on the CI tool itself). In either of these cases they'll need `kubeSecrets.create: true`